src/app/technical-record/technical-record.component.spec.ts

```diff
@@ -1,59 +1,149 @@
-import { HttpClientTestingModule } from '@angular/common/http/testing';
-import { CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
 import { ComponentFixture, getTestBed, TestBed } from '@angular/core/testing';
+import { CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
 import { FormsModule, ReactiveFormsModule } from '@angular/forms';
 import { MatDialog, MatDialogModule } from '@angular/material/dialog';
 import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
-import { RouterTestingModule } from '@angular/router/testing';
-import { MaterialModule } from '@app/material.module';
-import { SharedModule } from '@app/shared/shared.module';
-import { appReducers } from '@app/store/reducers/app.reducers';
-import { IAppState } from '@app/store/state/app.state';
-import { Store, StoreModule, INITIAL_STATE } from '@ngrx/store';
-import { hot } from 'jasmine-marbles';
-import { of, Subject } from 'rxjs';
 import { TechnicalRecordComponent } from './technical-record.component';
+import { TechRecordHelperService } from '@app/technical-record/tech-record-helper.service';
+import { TechnicalRecordValuesMapper } from './technical-record-value.mapper';
+
+import { TechRecord } from './../models/tech-record.model';
+import { MetaData } from '@app/models/meta-data';
+import {
+  VehicleTechRecordModel,
+  VehicleTechRecordEdit
+} from '@app/models/vehicle-tech-record.model';
+import { TestResultModel } from '@app/models/test-result.model';
+
+//to be moved to TESTING_UTILS
+const mockedActiveRecord = {
```

Comment:
👍
I think we might already have a few utils functions available from TESTING_UTILS so we avoid duplication.